common/metrics/config.go

```diff
@@ -475,25 +480,24 @@ func newPrometheusScope(
 
 // MetricsHandlerFromConfig is used at startup to construct a MetricsHandler
 func MetricsHandlerFromConfig(logger log.Logger, c *Config) (Handler, error) {
-	if c == nil {
+	if c == nil || c.Prometheus == nil {
```

Comment:
This is existing behavior, when creating tally handler, `NewScope` will return a noop scope if c.Prometheus is nil.